shared/metrics/src/main/java/io/pravega/shared/MetricsTags.java

```diff
@@ -36,6 +36,12 @@
      */
     private static final int TRANSACTION_ID_LENGTH = 2 * TRANSACTION_PART_LENGTH;
 
+    // For table segment name parsing, will depend on StreamSegmentNameUtils.
```

Comment:
I don't know if this is the place to set these constants. Please, keep an eye to PR #3629 and consider using (or even extending) the logic defined at `StreamSegmentNameUtils.java`. We need to avoid duplicating constants and naming patterns that already defined in one place.